var/spack/repos/spack_repo/builtin/packages/geomodel/package.py

```diff
@@ -65,6 +67,9 @@ class Geomodel(CMakePackage):
         description="Use the specified C++ standard when building",
     )
 
+    # GeoModel 6.11 drops support for C++17.
+    conflicts("cxxstd=20", when="@6.11:")
```

Comment:
Should this be `17`?
```suggestion
    # GeoModel 6.11 drops support for C++17.
    conflicts("cxxstd=17", when="@6.11:")
```

Reply:
Absolutely, thanks.

Reply:
I think this still needs to be applied. It is now choosing geomodel 6.10 in CI to be able to satisfy cxxstd=20. See https://gitlab.spack.io/spack/spack/-/jobs/16916871#L2143.

Reply:
What on earth I would have sworn that I made that change in this commit... Must be a case of Monday syndrome? :confused: